src/Buckle/Repl/Repl.cs

```diff
@@ -920,16 +924,22 @@ private sealed class SubmissionView {
         var parameters = command.method.GetParameters();
 
         if (args.Count != parameters.Length) {
-            var parameterNames = string.Join(" ", parameters.Select(p => $"<{p.Name}>"));
-            handle.diagnostics.Push(
-                new BelteDiagnostic(Repl.Diagnostics.Error.WrongArgumentCount(command.name, parameterNames)));
```

Comment:
This function generates ascii-text interpretations of REPL command declarations
e.g.
[MetaCommand("saveToFile", "Save previous <count> submissions to <path>")] -> "#saveToFile <path> <count=1>"

The code change here is supported default values for parameters ("<count>" -> "<count=1>")